libs/sdk-ui-tests-e2e/scenarios/src/components/Scenarios/Dashboard/DashboardTableTransposeScenario.tsx

```diff
@@ -0,0 +1,11 @@
+// (C) 2023 GoodData Corporation
+import React from "react";
+import { Dashboard } from "@gooddata/sdk-ui-dashboard";
+import { idRef } from "@gooddata/sdk-model";
+import { Dashboards } from "../../../../../reference_workspace/workspace_objects/goodsales/current_reference_workspace_objects_tiger";
+
+const dashboardRef = idRef(Dashboards.KDWithTableTranspose);
```

Comment:
I don't see this KD in the object "../../../../../reference_workspace/workspace_objects/goodsales/current_reference_workspace_objects_tiger". Did you define it somewhere?

Reply:
I added it on tiger_metadata_extension.js, because this test run intergrated checklist so i think it not necessary to update new current_reference_workspace_objects_tiger :)